src/adapters/beachfront.js

```diff
@@ -14,7 +14,7 @@ function BeachfrontAdapter() {
     const bids = bidRequests.bids || [];
     bids.forEach(function(bid) {
       var bidRequest = getBidRequest(bid);
-      var RTBDataParams = prepareAndSaveRTBRequestParams(bid);
+      var RTBDataParams = prepareAndSaveRTBRequestParams(bidRequest);
```

Comment:
`bid.params` should exist. I think the confusion here is that `bid` and `bidRequest` are referencing the same object. I can revert that line if it makes it more clear.